src/js/themes/hpe.js

```diff
@@ -202,6 +207,33 @@ export const hpe = deepFreeze({
     },
   },
   button: {
+    cta: {
+      background: {
+        color: 'brand',
+      },
+      border: undefined,
+      color: 'white', // 'text-strong',
+      font: {
+        weight: 700,
+      },
+      extend: props => {
```

Comment:
Comment for clarity.

```suggestion
     // TODO: Enhance Grommet to allow theme to set `button.{kind}.icon` which is an SVG.
     // With this enhancement, the `extend` property definition should be able to be removed.
      extend: props => {
```